src/components/config_parser.cpp

```diff
@@ -193,12 +193,16 @@ line_type config_parser::get_line_type(const string& line) {
     case '#':
       return line_type::COMMENT;
 
-    default:
+    default: {
+      if (string_util::contains(line, "﻿"))
+        throw syntax_error("The file encoding is not supported. Please use UTF-8");
```

Comment:
I think we should make this more specific. Since we only test for the UTF-8 encoding of the BOM, something like `This config file uses UTF-8 with BOM, which is not supported. Please use plain UTF-8 without BOM.` would make it clearer.

Reply:
Fixed in dd65b76